Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Emission' do
     pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-OSSUIFonts.git"
   end
 
-  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket']
+  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket', 'RCTImage']
```

Comment:
Cool. I'll start working on that. In the meantime, I'll make this pr mergeable by removing the chevron image for now.